content/en-US/api/USBEndpoint/USBEndpoint.md

```diff
@@ -0,0 +1,73 @@
+---
+recipe: api-feature
+title: 'USBEndpoint'
+mdn_url: /en-US/docs/Web/API/USBEndpoint
+specifications: https://wicg.github.io/webusb/#endpoints
+browser_compatibility: api.USBEndpoint
+---
+
+## Description
+
+The `USBEndpoint` interface of the WebUSB API provides information about an endpoint provided by the USB device. An endpoint represents a unidirectional data stream into or out of the device. Endpoints come in three flavors,
```

Comment:
Where's the three flavors?

Maybe say 'types' instead of 'flavors'. I would be ok with this word in a blog post, but for a technical reference it's a little colloquial.